app/services/validators/validation-error-messages.js

```diff
@@ -25,5 +25,6 @@ module.exports = {
   getMadeClaimForPrisonerBeforeIsRequired: 'You must confirm if you have made a claim before',
   getNoUpdatesMade: 'No updates were made, please ensure all documents are uploaded for your application to be processed or send a message to your caseworker. Otherwise press cancel to return to your claims',
   getInvalidReference: 'Reference is invalid',
-  getMalwareDetected: 'We have detected a virus in this file and can not accept it. Please upload a virus free version.'
+  getMalwareDetected: 'We have detected a virus in this file and can not accept it. Please upload a virus free version.',
+  getNoExpensesClaimedFor: function () { return 'All expenses were removed, add an expense to continue' }
```

Comment:
Check with Chris, if you haven't already that this error message is appropriate.